src/main/java/com/training/utilities/UuidValidator.java

```diff
@@ -0,0 +1,12 @@
+package com.training.utilities;
+
+import java.util.regex.Pattern;
+
+public class UuidValidator {
+
+    public boolean isValidUUID(String accountId) {
+        return accountId==null ? false:
+                Pattern.compile("^[{]?[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[}]?$")
+                        .matcher(accountId).matches();
```

Comment:
This can be simplified as :
`accountId != null && Pattern.compile("^[{]?[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[}]?$")
                .matcher(accountId).matches();`

Reply:
I had done this, but didn't commit coz pranav had asked for ternary operator